tests/v3_api/test_nfs.py

```diff
@@ -0,0 +1,213 @@
+import pytest
+from .common import *  # NOQA
+
+namespace = {"p_client": None,
+             "ns": None,
+             "cluster": None,
+             "project": None,
+             "pv": None,
+             "pvc": None}
+
+PV_SERVER = os.environ.get("PV_SERVER", "")
+
+# the name of the file and its content that will be saved to the nfs
+test_file_name = "test-wl-1.txt"
+test_content = ""
+
+
+def write_content_to_file(pod):
```

Comment:
Can we have the contents to be passed as argument to the function and move this to common.py